transport/protocol/tests/common/mod.rs

```diff
@@ -138,18 +137,22 @@ pub async fn create_minimal_topology(dbs: &mut Vec<HoprDb>) -> anyhow::Result<()
     Ok(())
 }
 
+#[allow(dead_code)]
```

Comment:
We should remove all this deadcode and if ticket aggregation needs to be re-added, it will be done from scratch as a Session protocol.